filesystem/log.cpp

```diff
@@ -0,0 +1,12 @@
+#include "log.h"
+#include "../common/log.h"
+#include <cstdio>
+
+int fuse_levels_to_internal[] = {WARN, WARN, ERROR, ERROR, WARN, INFO, INFO, DEBUG};
```

Comment:
Can we add here a comment with info from fuse_log_level enum? (just so it's clear when looking at that part what corresponding values are).
`enum fuse_log_level {
	FUSE_LOG_EMERG,
	FUSE_LOG_ALERT,
	FUSE_LOG_CRIT,
	FUSE_LOG_ERR,
	FUSE_LOG_WARNING,
	FUSE_LOG_NOTICE,
	FUSE_LOG_INFO,
	FUSE_LOG_DEBUG
};`

Why the corresponding value for fuse ALERT and fuse EMERG is WARN?

Reply:
Good point, I thought about the comment but finallay didn't added. I also fixed the EMERG and WARN levels.